graph-chat.md

```diff
@@ -30,20 +30,18 @@ The following symbols are used below:
   - `CAi` - Alice's simplex connection number `i` (out of `n`) allowing Bob to send messages to Alice.
   - `CBi` - Bob's simplex connection number `i` (out of `n`) allowing Alice to send messages to Bob.
 - keys created for Alice's connections:
+  - `RUAi` - server-generated recipient connection URI of `CAi` (to be used by Alice to retrieve messages).
   - `EKAi` - Alice's assymetric key pair used:
     - by Bob to encrypt and by Alice to decrypt messages from Bob sent via `CAi`.
     - by Alice to sign and Bob to verify messages from Alice sent via `CBi`.
-  - `IDAi` - client-generated ID of connection `CAi`
-  - `RUAi` - server-generated recipient connection URI of `CAi` (to be used by Alice to retrieve messages).
   - `SUAi` - server-generated sender connection URI of `CAi` (to be used by Bob to send messages).
   - `RKAi` - Alice's recipient key of `CAi`.
   - `SKAi` - Bob's sender key of `CAi`.
 - keys created for Bob's connections
+  - `RUBi` - server-generated recipient connection URI of `CBi` (to be used by Bob to retrive messages).
   - `EKBi` - Bob's assymetric key pair used for:
     - Alice to encrypt and Bob to decrypt messages from Alice sent via `CBi`.
     - Bob to sign and Alice to verify messages from Bob sent via `CAi`.
-  - `IDBi` - client-generated ID of connection `CBi`
-  - `RUBi` - server-generated recipient connection URI of `CBi` (to be used by Bob to retrive messages).
   - `SUBi` - server-generated sender connection URI of `CBi` (to be used by Alice to send messages).
```

Comment:
shouldn't `RUAi`, `SUAi`, `RUBi` and `SUBi` be moved under a separate section / sections (e.g. "connection URIs" or "[Alice's / Bob's] connection URIs")? I mean they're not technically keys like `EKs`, `RKs` and `SKs`, aren't they?

Reply:
it can be changed together with #6